projects/filet/index.js

```diff
@@ -1,10 +1,91 @@
-const { get } = require('../helper/http')
 
-async function fetch() {
-  const resp = await get('https://api.filet.finance/pledge/ext/tx/pledgeTxAll')
-  return resp.data.tvl;
+const abi = require("./abi.js") 
+const { fetchURL } = require('../helper/utils')
+
+
+const BigNumber = require("bignumber.js")
+const { getExports } = require('../helper/heroku-api')
+const { nullAddress } = require("../helper/tokenMapping");
+const { get } = require("../helper/http");
+const {utils} = require("ethers");
+
+//bsc staking con
+const filetStakingCon_BSC = "0x9c821defD3BBb07C5c786C3bB039051364Fa6F39";
+const filetStakingConFlexible_BSC = "0xE989ef3118C685603B86123569340438ab670505";
+//bsc FILE con
+const filetFILECon_BSC = "0xA8F19840e08dC3f134FF45062b953a2B14E02F5A"
+const filetStorageCon_FVM = "0xFca90089d63D099c80A4ae91edCdfC904c2afa8e";
+const filetNativeCon = "0xBfd5BB742940a5b059710F4d1216db1F1287F4e0";
+
+const filetStakingCon_HECO = "0x62C7aaCdCCEc32A2E33d5fa535B374f8a4a42566";
+const filetStakingConFlexible_HECO = "0x64F23Eb153D7496Fc31C1b38a41068995354773E"
+
+const poolOnFVM180 = 4010
+const poolOnFVM360 = 4011
+
+// filet 
+const filetAPI = "https://api.filet.finance/pledge/ext/tx/pledgeTxAll"
```

Comment:
unable to test, this api is returning 502 :(

Reply:
Sorry for inconvenient
Our dev teams have an upgrading plan for server, It works normally now,
Please try again